dev/com.ibm.websphere.appserver.features/visibility/public/ejbLite-4.0/resources/l10n/com.ibm.websphere.appserver.ejbLite-4.0.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables support for Jakarta Enterprise Beans written to the Enterprise Beans Lite subset of the Jakarta Enterprise Beans 4.0 specification.
```

Comment:
suggested edit:
This feature enables support for Jakarta Enterprise Beans that are written to the Enterprise Beans Lite subset of the Jakarta Enterprise Beans 4.0 specification.

Reply:
@dmuelle Updated as suggested